_includes/simple-card.html

```diff
@@ -0,0 +1,31 @@
+<ul class="usa-card-group padding-top-2">
+  {% assign list = site.[include.collection] %}
+  {% for item in list %}
+    <li class="tablet:grid-col-{{include.colwidth}} usa-card">
+      <div class="usa-card__container">
+        <div>
+          {% if item.title != "" %}
```

Comment:
[dust] -- does an empty string pass the truthy check in Jekyll? 
```suggestion
          {% if item.title %}
```